stable/airflow/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 7.13.1
+version: 7.13.2
```

Comment:
If you add a value, yo must bump a minor version, and create documentation in UGRADE.md

So in this case `7.14.0`